app/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,7 @@
   <string name="topic_play_chapter_index">%s.</string>
   <string name="chapter_name">Chapter %s: %s</string>
   <string name="chapter_completed">Chapter %s: %s is completed.</string>
+  <string name="chapter_prerequisite">Complete Chapter <b>%s: %s</b> to unlock this chapter.</string>
```

Comment:
Some of these strings need some work to be made more clear, but I think we can add more context for this string in particular. Perhaps: ``chapter_prerequisite_title_label`` might be clearer. I think my issue with the current label is it's a bit vague--we want to try to contextualize it to this view. However, I'm having slight difficulties at the moment coming up with a clearer name. Any thoughts?

Reply:
Renamed to `chapter_prerequisite_title_label`.